src/components/Button/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ icon, children }) => {
+  return (
```

Comment:
Button should have  at least `type`, `onClick` properties

Reply:
I will delete this component, we don't actually need it.

Reply:
And why do you need this component in PR if you are not using it? =)